charts/cla/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
This file ingress definition is too configurable. We should use a more ad-hoc configuration, like https://github.com/w3f/kusama-faucet/blob/master/charts/kusama-faucet/templates/ingress.yaml